docs/maps/editor/entry-exit.md

```diff
@@ -1,7 +1,47 @@
-# Creating entries and exits
+# Creating start and exits
 
-TODO: entries and exits
+## Create a start area
+In order to let user spawn on your map in a defined area you will need to create a start area.
+To do that, you must follow those instructions :
+1. Open the map editor.
+2. Access the "area map editor" section.
+3. Create an area by clicking on the map directly or by selecting a zone.
+4. Click on the "start" icon.
+<div class="row">
+    <div class="col">
+        <img src="../images/editor/start_area_1.png" class="figure-img img-fluid rounded" alt="" />
```

Comment:
Same comment as yesterday (sorry): it would be better to have a screenshot on a real map, with something looking like an exit so that readers can project themselves. Ask Valdo for such a map :)